composer/functions/composer_storage_trigger_test.py

```diff
@@ -11,12 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import composer_storage_trigger
```

Comment:
this change needs to be reverted and lint will then pass

Reply:
Interesting. With my local nox run I got:

```
./functions/composer_storage_trigger_test.py:18:1: I100 Import statements are in the wrong order. 'import composer_storage_trigger' should be before 'import pytest'
import composer_storage_trigger
```

That is why I moved the import. But if this fails ci/cl I am reverting the change.